src/components/program-progress/SubscriptionExpiringWarningModal.jsx

```diff
@@ -37,7 +37,7 @@ const SubscriptionExpirationWarningModal = ({
         If you plan to complete the program, please {renderContactText()} to ensure your subscription access is renewed.
       </p>
       <i>
-        Access expires: {dayjs(subscriptionPlan.expirationDate).format('MMMM Do, YYYY')}.
+        Access expires: {dayjs(subscriptionPlan?.expirationDate).format('MMMM Do, YYYY')}.
```

Comment:
[clarification] When would this render without `subscriptionPlan` existing? If `subscriptionPlan` truly can be missing here, I believe this would display that "Access expires: <TODAYS DATE>" since `dayjs(undefined)` would return the current timestamp.

Reply:
Figured out the issue here. There was conditional blocking the rendering of this component that was left out of this PR with a check around truthy subscription plans. 

``` jsx
  if (
    !subscriptionPlan || !(showExpirationNotifications || courseEndDate > subscriptionPlan?.expirationDate)
  ) {
    return null;
  }
 ```